src/main/java/io/xpring/common/CommonUtils.java

```diff
@@ -0,0 +1,35 @@
+package io.xpring.common;
```

Comment:
These things exist in the poorly named XrpUtils.java: https://github.com/xpring-eng/xpring4j/blob/5c07b0f664493777af211379f748ba2dd5312be0/src/main/java/io/xpring/xrpl/Utils.java.

Agree this should be common. Can we deprecate those methods?

Reply:
deprecated